tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial
+title: "International Flutter Apps"
+
+permalink: /tutorials/international/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* How to track the device's locale (the user's preferred language).
+* How to manage locale-specific app values.
+* How to extend the locales that Flutter supports.
+
+</div>
+
+<a name="How To Write International Flutter Applications"></a>
+## How To Write International Flutter Applications
+
+If your app might be deployed to users who speak another language then
+you'll need to "internationalize" it. That means you'll need to write
+the app in a way that makes it possible to "localize" values like text
+and layouts for each language or "locale" that the app
+supports. Flutter provides widgets and classes that help with
+internationalization and the Flutter libraries themselves are
+internationalized.
+
+### The `Locale` class and the `Localizations` Widget
+
+The [`Locale`](https://docs.flutter.io/flutter/dart-ui/Locale-class.html)
+class is used to identify the user's language. Mobile devices support
+setting the locale for all applications, usually via a system settings
+menu. Internationalized apps respond by redisplaying values that are
```

Comment:
redisplaying? that sounds like "display a value, then display that value again". Maybe "displaying" or "display"...